sys-fs/duf/duf-0.8.1.ebuild

```diff
@@ -7,8 +7,10 @@ inherit go-module
 
 DESCRIPTION="Disk Usage/Free Utility - a better 'df' alternative"
 HOMEPAGE="https://github.com/muesli/duf"
-SRC_URI="https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz"
```

Comment:
Would you mind keeping this format (+=)? It's a bit easier for my scripts when i do dep tarballs myself.

Reply:
Sure thing